### ISSUE TITLE
Assertion `cst->getNumOperands() == type->num_fields()' failed.

### ISSUE THREAD

**Issue by practicalswift** [NASA-SW-VnV/ikos#67]
```
$ cat assertion-cst-getnumoperands-type-num-fields-failed.c 
struct {
  int a[0];
  struct {
    int b
  };
} c = {{}, 6};
$ ikos assertion-cst-getnumoperands-type-num-fields-failed.c
[*] Compiling assertion-cst-getnumoperands-type-num-fields-failed.c
…
2 warnings generated.
[*] Running ikos preprocessor
[*] Running ikos analyzer
[!] ikos was compiled in debug mode, the analysis might be slow
[*] Translating LLVM bitcode to AR
ikos-analyzer: /.../frontend/llvm/src/import/constant.cpp:262: ikos::ar::StructConstant* ikos::frontend::import::ConstantImporter::translate_constant_struct(llvm::ConstantStruct*, ikos::ar::StructType*, ikos::ar::BasicBlock*, ikos::frontend::import::ConstantImporter::ConstantExpressionList&): 
  Assertion `cst->getNumOperands() == type->num_fields()' failed.
…
ikos: error: exited with signal SIGIOT
```

**Comment by arthaud** [comment]
Thanks for the bug report.

This is because the structure type in IKOS is a map from offset to type.
In that case `a` and `b` have the same offset (i.e, 0) so the field `a` gets removed.

I need to change the structure type to allow different fields with the same offset. We cannot just ignore the field `a` since it might add alignment requirements.

**Comment by arthaud** [comment]
This should be fixed by 1c12398390fdaba4797ea384ec9ec1172c2d9c7b. Feel free to reopen the issue if the issue is still here.

**Comment by practicalswift** [comment]
Confirming that I'm not longer encountering this issue. Thanks!
